src/components/EditPost.tsx

```diff
@@ -24,6 +24,11 @@ interface IUpdateResponse {
   success: boolean;
   message: string;
 }
+
+interface IFormError {
+  message: string;
+  path: string[];
```

Comment:
make path array of string or number, current state might a fail at build time.

Reply:
Alright

Reply:
Made the changes

Reply:
The build still fails. Will check the cause(s) and run a build version before push.